packages/xrpl/src/models/ledger/Escrow.ts

```diff
@@ -19,8 +21,8 @@ export default interface Escrow extends BaseLedgerEntry {
    * successful.
```

Comment:
Please edit the other docstrings in this file (e.g. the interface docstring) to remove references to XRP. Ditto with the other models.

Reply:
This is an outdated commit... I will check everything again though